apps/fluent-tester/src/RNTester/TestComponents/PersonaCoin/styles.ts

```diff
@@ -1,14 +1,11 @@
-import { StyleSheet } from 'react-native';
+import {StyleSheet} from 'react-native';
 
-export const styles = StyleSheet.create({
-  oneCoin: {
-    margin: 8
-  }
-});
+export const styles = StyleSheet.create({oneCoin: {margin: 8}});
 
 export const satyaPhotoUrl =
-  'https://www.microsoft.com/en-us/CMSImages/satya.jpg?version=0881eb71-4942-b627-d602-84c832b8a0b6&amp;CollectionId=1b46ce2d-c90d-421e-94f1-cfb6bc6ef6ec';
+    'https://upload.wikimedia.org/wikipedia/commons/5/53/Satya_bookcover-print.jpg';
 
-export const steveBallmerPhotoUrl = 'http://www.rmndigital.com/wp-content/uploads/2012/11/stevems.jpg';
+export const steveBallmerPhotoUrl =
+    'https://www.rmndigital.com/wp-content/uploads/2012/11/stevems.jpg';
```

Comment:
Perhaps we would want to upload these to our CDN so we don't have to rely on external links?

Reply:
Outside of scope though